iroh-gateway/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "iroh-gateway"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0/MIT"
+readme = "README.md"
+description = "IPFS gateway"
+repository = "https://github.com/dignifiedquire/iroh"
+
+[dependencies]
+cid = "0.8.4"
+tokio = { version = "1.17.0", features = ["macros", "rt-multi-thread", "process"] }
+axum = "0.5.1"
+clap = { version = "3.1.8", features = ["derive"] }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0.78"
+tower = { version = "0.4", features = ["util", "timeout", "load-shed", "limit"] }
+mime_guess = "2.0.4"
+
+#[dev-dependencies]
```

Comment:
is this a `dev-dependency` or not?

Reply:
axum-macros? yeag totally. Unwraps messy generated/wrapper handler code to provide better debug messages

Reply:
why is this commented out then?

Reply:
...